### PR TITLE
Localflavor Sri Lanka: Support for Sri Lanka added in django-localflavor

### PR DESCRIPTION
**Pull request by skdishansachin**
**Support for Sri Lanka Added as requested by this issue https://github.com/django/django-localflavor/issues/491 **

**What Fields are added ?**

- [x] Field for provinces of Sri Lanka.
- [x] Field for districts of Sri Lanka.
- [x] Field for Postal Code of Sri Lanka.

**All Changes**

- [x] Add an entry to the docs/changelog.rst describing the change.

- [x] Add an entry for your name in the docs/authors.rst file if it's not
      already there.

**New Fields Only**

- [x] Prefix the country code to all fields.

- [x] Field names should be easily understood by developers from the target
      localflavor country. This means that English translations are usually
      not the best name unless it's for something standard like postal code,
      tax / VAT ID etc.

- [x] Prefer '<country code>PostalCodeField' for postal codes as it's
      international English; ZipCode is a term specific to the United
      States postal system.

- [x] Add meaningful tests. 100% test coverage is not required but all
      validation edge cases should be covered.

- [x] Add `.. versionadded:: <next-version>` comment markers to new
      localflavors.

- [x] Add documentation for all fields.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -27,6 +27,8 @@ Other changes:
 
 New flavors:
 
+- Sri Lanka LocalFlavor: Support for Sri Lanka added
+  (`gh-493 <https://github.com/django/django-localflavor/pull/493>`_).
```
Comment (claudep) [inline docs/changelog.rst:31]: You should not add any entry in notes of an already-released version. Please move that note above in the unreleased section.

For the rest, it looks rather good, and I hope @benkonrath will have the opportunity to make a detailed review during the next weeks. Thanks!

Comment (dishansa) [reply docs/changelog.rst:31]: Okay, I will make that change. I wasn't aware of it before.

**Comment by claudep** [comment]
@benkonrath Do you think you could find some time in the near future to review this request?

**Comment by benkonrath** [comment]
@claudep Yeah. I'll try to dedicate a few hours to going through the PR backlog a bit in the next day or two. Thanks for the reminder.

**Comment by claudep** [comment]
@skdishansachin Please fix the test failures!

**Comment by skdishansachin** [comment]
Okay I will fix those test failures @claudep

**Comment by benkonrath** [comment]
@skdishansachin Please fix the tests and then I'll take a look at this PR. I don't know why I have to continually approve the test runs but I'll be quicker about doing this now that I know that they need to approved more than once.

**Comment by skdishansachin** [comment]
@benkonrath  @claudep Hey, sorry about sending that review request before fixing the issues properly. I'll check it soon when I have time. Thanks!

**Comment by benkonrath** [comment]
No problem. There's no direct feedback since I have to approve and run the tests in github CI... and I haven't been on top of that. That makes it harder for you to know that things aren't working correctly. This PR is on my radar now and hopefully we can make some progress soon. :rocket:

**Comment by skdishansachin** [comment]
@benkonrath Oh sounds good to me. :+1:

**Comment by benkonrath** [comment]
I fixed the tests and implemented a few cleanups / improvements. I think it's good to go.

**Comment by skdishansachin** [comment]
@benkonrath  Thanks for all the tweaks you made to get this working! Sorry for leaving this pull request hanging. I got caught up in my studies and exams. I checked out the changes you made, and they look great. I really appreciate it!

**Comment by benkonrath** [comment]
@skdishansachin No problem. Thanks for looking over my changes.
